src/mpid/ch4/include/mpidch4.h

```diff
@@ -18,6 +18,8 @@
  */
 #define MPIDI_CH4I_API(rc,fcnname,...)            \
   MPL_STATIC_INLINE_PREFIX rc MPID_##fcnname(__VA_ARGS__) MPL_STATIC_INLINE_SUFFIX
+#define MPIDI_CH4I_API_NOINLINE(rc,fcnname,...)            \
+  rc MPID_##fcnname(__VA_ARGS__)
```

Comment:
I would define this in a separate commit.

Reply:
I think I will squash the whole thing into one commit for CH4, another for each netmod/shm/generic. Probably leave this in the CH4 one for atomicity.